app/ForkedStar.tsx

```diff
@@ -0,0 +1,76 @@
+import octokit from "@/lib/octokit";
+import { env } from "@/env.mjs";
+import { GoRepoForked } from "react-icons/go";
+import { GoStar } from "react-icons/go";
+import { GoPeople } from "react-icons/go";
+import { getContributors } from "@/lib/api";
+
+const GITHUB_ORG: string = env.NEXT_PUBLIC_GITHUB_ORG;
+
+export default async function ForkedStar() {
+  const count = await getData().then((data) => {
+    return data.reduce(
+      (acc, ls) => {
+        if (ls.forks) {
+          acc.forkCount += ls.forks;
+        }
+        if (ls.stargazers_count) {
+          acc.starCount += ls.stargazers_count;
+        }
+        return acc;
+      },
+      { forkCount: 0, starCount: 0 },
```

Comment:
Just some nitpicks...

```suggestion
        return {
          forks: acc.forks + ls.forks,
          stargazers_count: acc.stargazers_count + ls.stargazers_count,
        },
        { forks: 0, stargazers_count: 0 },
```